docs(change log): add change log for 4.3.9 (#891)
emqtt/emq-docs-cn: en_US/changes/changes-4.3.md

# Changes

## Version 4.3.9

*Release Date: 2021-11-02*

EMQ X 4.3.9 is released now, it mainly includes the following changes:

> **Bug fixes:**

Reviewer: ### Important fixes

### Minor fixes

Author: I changed to "Bug fixes (Important)"

> - Fix the issue that WebHook's TLS is unavailable

Reviewer: ```suggestion
- WebHook's HTTP client SSL configuration parse
```

  Github PR: [emqx#5696](https://github.com/emqx/emqx/pull/5696)

> - Fix the issue that MongoDB resources do not support domain names

Reviewer: ```suggestion
- MongoDB resources allow host names
```

  Github PR: [emqx#6035](https://github.com/emqx/emqx/pull/6035)

> - Fix the performance of ACL based on the built-in database

Reviewer: This is an enhancement

```suggestion
- Performance improvement for built-in database ACL (emqx_auth_mnesia)
```

  Github PR: [emqx#5885](https://github.com/emqx/emqx/pull/5885)

- Fix the issue that the authentication based on the built-in database incorrectly transcodes the HTTP request parameters

  Github PR: [emqx#5674](https://github.com/emqx/emqx/pull/5674)

> - Fixed the issue that the Client ID containing "\" characters could not be searched in a fuzzy manner

Reviewer: ```suggestion
- Allow "\" characters in client ID (which previously affected fuzzy lookup)
```

Author: The `\` character was previously supported, but it will affect fuzzy queries. The new description looks like we did not support client connections that contained `\` characters.

  Github PR: [emqx#5978](https://github.com/emqx/emqx/pull/5978)

- Fix the issue that variable byte integers may be larger than 4 bytes

  Github PR: [emqx#5826](https://github.com/emqx/emqx/pull/5826)

- Fix the issue that the rules cannot be modified in cluster

  Github PR: [emqx#5731](https://github.com/emqx/emqx/pull/5731)

> - Fix an issue where low priority queues may not be processed when messages are forwarded with different priorities

Reviewer: This is an enhancement.

```suggestion
- Introduced interleaving for priority queues (to avoid low priority queue stavation)
```

  Github PR: [emqx#5666](https://github.com/emqx/emqx/pull/5666)

> - Fix the issue that STOMP client cannot be displayed on the Dashboard page, and supports subscribing and unsubscribing operations on the page

Reviewer: ```suggestion
- Fixed STOMP clients view & management from dashboard
```

  Github PR: [emqx#6040](https://github.com/emqx/emqx/pull/6040)

- Fix the issue that the ID of the STOMP client may not be unique in the cluster

>   Github PR: [emqx#6040](https://github.com/emqx/emqx/pull/6040)

Reviewer: delete this item.

- Fix the authentication issue of STOMP gateway

>   Github PR: [emqx#6040](https://github.com/emqx/emqx/pull/6040)

Reviewer: delete this item

**Enhancement:**

> - Improve client kicking mechanism

Reviewer: ```suggestion
- Improve client kick (forced step-down)
```
